### PR TITLE
MinAccuracy, accurate siteCoverage

### PR DESCRIPTION
**Pull request by nlhepler**

### CODE REVIEW

**Comment by dalexander** [comment]
So technically the sitecoverage change is "off-target" of what we are trying to fix here, but let me review it anyway...

File: GenomicConsensus/arrow/utils.py [hunk 1]
```diff
@@ -342,51 +367,48 @@ def consensusForAlignments(refWindow, refSequence, alns, arrowConfig):
...
-
-    # TODO(lhepler, dalexander): propagate coverage around somehow
+        if not sufficientlyAccurate(mr, poaCss, arrowConfig.minAccuracy):
+            tpl = poaCss[mr.TemplateStart:mr.TemplateEnd]
+            if mr.Strand == cc.StrandEnum_FORWARD:
+                pass
+            elif mr.Strand == cc.StrandEnum_REVERSE:
+                tpl = reverseComplement(tpl)
+            else:
+                tpl = "NA"
```
Comment (dalexander) [inline GenomicConsensus/arrow/utils.py]: why would strand not be in {FWD, REV}?  Is these an "inactive evaluator" situation?

**Comment by dalexander** [comment]
OK Lance, please pull the sitecoverage changes out into a separate commit/PR, this could easily break things in an unforeseen way and it's not appropriate for S3.1

**Comment by nlhepler** [comment]
deprecating this one in favor of another sans siteCoverage fix

**Pull request closed without merging**